src/combobox/type.ts

```diff
@@ -30,9 +31,10 @@ export declare class Combobox extends KucBase {
   error: string;
   id: string;
   label: string;
-  value: string;
   disabled: boolean;
+  placeholder: string;
   requiredIcon: boolean;
+  value: string;
```

Comment:
Please sort the properties by group.
```suggestion
  placeholder: string;
  value: string;
  disabled: boolean;
  requiredIcon: boolean;
```